src/Abstractions/NexusMods.Abstractions.NexusWebApi/Types/NXMProtocolRegistrationTest.cs

```diff
@@ -0,0 +1,28 @@
+using JetBrains.Annotations;
+
+namespace NexusMods.Abstractions.NexusWebApi.Types;
+
+/// <summary>
+/// NXM protocol registration test.
+/// </summary>
+[PublicAPI]
+public class NXMProtocolRegistrationTest : NXMUrl
```

Comment:
Should this be called "check" rather than "test" to avoid overload with unit tests?